src/app/views/customers/batch-upload/batch-upload.controller.coffee

```diff
@@ -1,7 +1,16 @@
-@App.controller 'BatchUploadController', ($window) ->
+@App.controller 'BatchUploadController', ($window, $injector, MnoeCurrentUser) ->
   'ngInject'
   vm = this
 
+  MnoeCurrentUser.getUser().then( ->
+    vm.user = MnoeCurrentUser.user
+    if !vm.user.id?
+      toastr = $injector.get('toastr')
```

Comment:
You don't have to use the `$injector` as we're in a Controller, you can specify `toastr` as dependency and it'll get injected.
We're using the `$injector` for the other interceptor because we're in `config` block